### PR TITLE
added resources for web-dev , ml ,competitive programming

### PR DESCRIPTION
**Pull request by sanil011**
Please add the tag of hacktoberfest.
issue no[#17 ]
I have a question can I add paid courses of udemy , I will not add link just name the course

### CODE REVIEW

**Comment by aaryan2134** [comment]
image
Please correct the formatting issues
+

**Comment by aaryan2134** [comment]
Please update the contents as well

**Comment by sanil011** [comment]
> Please update the contents as well

udemy course??

**Comment by sanil011** [comment]
> image Please correct the formatting issues +

is it now ok?

**Comment by aaryan2134** [comment]
Great work!

**Comment by aaryan2134** [comment]
Please add the Web Development section in the index

File: README.md [hunk 1]
```diff
@@ -1,45 +1,51 @@
...
 ## Contents
+
 - [Engineering Resources](#engineering-resources)
   - [Contents](#contents)
   - [Introduction](#introduction)
   - [Student Offers <a name="student-programs"></a>](#student-offers-)
   - [CS Resources <a name="education"></a>](#cs-resources-)
   - [Machine Learning Resources <a name="machine-learning"></a>](#machine-learning-resources-)
   - [Competitive Programming <a name="competitive-programming"></a>](#competitive-programming-)
-
```
Comment (aaryan2134) [inline README.md:12]: Add web development section here and make sure it is pointing to the added section as well

**Comment by aaryan2134** [comment]
Great Work!
Thank you for your contribution

**Comment by sanil011** [comment]
hey let's connect

**Pull request merged**